src/record/record-handler.js

```diff
@@ -288,7 +291,8 @@ RecordHandler.prototype._update = function (socketWrapper, message) {
   }
 
   if (this._transitions[recordName] && this._transitions[recordName].hasVersion(version)) {
-    this._transitions[recordName].sendVersionExists(socketWrapper, version)
+    const tempUpdate = { message, version, sender: socketWrapper }
```

Comment:
inline as in:

`
this._transitions[recordName].sendVersionExists({
  message,
  version,
  sender: socketWrapper
})
`